studio/components/interfaces/Settings/Vault/Keys/EncryptionKeysManagement.tsx

```diff
@@ -329,11 +329,7 @@ const EncryptionKeysManagement = () => {
                     Provide a name for your key for easier identification.
                   </p>
                   <div className="space-y-4 pb-4">
-                    <Input
-                      id="name"
-                      label="Key Name"
-                      descriptionText="Keys are of standard UUID types."
```

Comment:
> descriptionText="Keys are of standard UUID types."

no longer relevant?

Reply:
yeah this one's no relevant - there's no point informing users what the key types are